package.json

```diff
@@ -34,6 +34,8 @@
     "react": "^0.13.0"
   },
   "dependencies": {
+    "create-react-class": "^15.5.1",
+    "prop-types": "^15.5.6",
```

Comment:
> but might not use prop-types or create-react-class
I guess it's true

> Can you explain why you think they should be peerDependencies?

I just don't think that this lib is big enough to be "batteries included" type.

____

Anyways I guess @gaearon [does not have time](https://github.com/gaearon/react-document-title/pull/37#issuecomment-289696913) or ambitions to maintain this project, so it's time to move on.